server/manager/src/main/java/org/apache/accumulo/manager/upgrade/UpgradeCoordinator.java

```diff
@@ -152,8 +155,13 @@ public synchronized void upgradeZookeeper(ServerContext context,
         abortIfFateTransactions(context);
 
         for (int v = currentVersion; v < AccumuloDataVersion.get(); v++) {
-          log.info("Upgrading Zookeeper from data version {}", v);
-          upgraders.get(v).upgradeZookeeper(context);
+          log.info("Upgrading Zookeeper from data version {} target version {}", v,
+              AccumuloDataVersion.get());
```

Comment:
I think this log message change could be misleading. These are incremental upgrades, but the target version is just the final version. For example, the message may imply that it is upgrading from 5 to 9, but is really doing 5 to 6, then 6 to 7, etc. until it gets to 8 to 9.

Reply:
Addressed with b3ca1d9b65